GiniVision/Classes/Core/Gallery/AlbumsPickerTableViewCell.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  AlbumsPickerTableViewCell.swift
+//  GiniVision
+//
+//  Created by Enrique del Pozo Gómez on 2/27/18.
+//
+
+import Foundation
+
+final class AlbumsPickerTableViewCell: UITableViewCell {
+    
+    static let identifier = "AlbumsPickerTableViewCellIdentifier"
+    static let height: CGFloat = 90.0
+    let padding = UIEdgeInsets(top: 10, left: 10, bottom: 10, right: 10)
+    
+    lazy var albumAccesoryView: UIButton = {
+        let button = UIButton(frame: .zero)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.setImage(UIImageNamedPreferred(named: "disclosureIndicator"), for: .normal)
```

Comment:
Does it make sense to replace these strings with an enum (not in this PR).

Reply:
Really good idea, added to the Github issues.